### PR TITLE
Ignore bugs filed by the wpt-sync bot 

### PR DESCRIPTION
**Pull request by marco-c**
Applies on top of #374.

### CODE REVIEW

File: auto_nag/scripts/regression.py [hunk 1]
```diff
@@ -50,6 +50,10 @@ def get_bz_params(self, date):
             'emailtype1': 'notequals',
             'email1': 'intermittent-bug-filer@mozilla.bugs',
             'emailreporter1': 1,
+            # And by the wpt-sync bot.
```
Comment (calixteman) [inline auto_nag/scripts/regression.py]: I'd prefer to have a black-list of emails in config/tools.json.

Comment (marco-c) [reply auto_nag/scripts/regression.py]: Done

File: auto_nag/scripts/regression.py [hunk 2]
```diff
@@ -46,16 +47,14 @@ def get_bz_params(self, date):
...
-            'emailtype1': 'notequals',
-            'email1': 'intermittent-bug-filer@mozilla.bugs',
-            'emailreporter1': 1,
-            # And by the wpt-sync bot.
-            'emailtype2': 'notequals',
-            'email2': 'wptsync@mozilla.bugs',
-            'emailreporter2': 1,
         }
 
+        for i, email in enumerate(utils.get_config(self.name(), 'reporter_blacklist')):
```
Comment (calixteman) [inline auto_nag/scripts/regression.py]: 'f6': 'reporter', 'o6': 'nowords', 'v6': ','.join(blacklist) should be better

Comment (marco-c) [reply auto_nag/scripts/regression.py]: Done

**Comment by sylvestre** [comment]
@marco-c do you remember why we did that?

**Comment by marco-c** [comment]
This was changing the 'regression' script config rather than the no_crashes one, no_crashes was ignoring intermittent-bot since the beginning (#202).

For 'regression', we did this because:
1) wptsync never files bugs which are regressions (it only files bugs tracking web-platform-tests updates);
2) the intermittent bot sets the regression keyword by itself.

**Pull request merged**